alvr/server/src/connection.rs

```diff
@@ -643,9 +644,30 @@ fn try_connect(mut client_ips: HashMap<IpAddr, String>) -> ConResult {
     };
 
     let tracking_manager = Arc::new(Mutex::new(TrackingManager::new()));
+    let hand_gesture_manager = Arc::new(Mutex::new(HandGestureManager::new()));
 
     let tracking_receive_thread = thread::spawn({
         let tracking_manager = Arc::clone(&tracking_manager);
+        let hand_gesture_manager = Arc::clone(&hand_gesture_manager);
+
+        let mut gestures_button_mapping_manager = if let Switch::Enabled(config) =
+            &SERVER_DATA_MANAGER.read().settings().headset.controllers
+        {
+            if let Switch::Enabled(_) = &config.hand_tracking.use_gestures {
+                Some(ButtonMappingManager::new_automatic(
+                    &CONTROLLER_PROFILE_INFO
+                        .get(&alvr_common::hash_string(QUEST_CONTROLLER_PROFILE_PATH))
+                        .unwrap()
+                        .button_set,
```

Comment:
This is wrong, you are supposed to create your own button set. The set of button you should register are:
* A/X click
* B/Y click
* trigger click
* grip value
* joystick X/Y
* optionally joystick click

for left and right hands.
The Quest controller has also touch inputs and if you register them for hand gestures the button mapping system will not generate the correct bindings. If you omit touch inputs, the touch events will be raised automatically on click.

Reply:
You can define the button set in a `static Lazy<>` variable in `hand_grstures.rs`. In the future we may just bind the buttons the user wants to use.

Reply:
> The Quest controller has also touch inputs and if you register them for hand gestures the button mapping system will not generate the correct bindings. If you omit touch inputs, the touch events will be raised automatically on click.

With this, since I'm sending touch events, should I include touch binds in the mapping thing?

Reply:
Yes, but I don't think you should detect touch events at all with hand tracking.

Reply:
For now I've left the touch events in as they are causing no issues. Removing them would be quite easy in future though if something goes wrong with them.